README.md

```diff
@@ -138,6 +143,41 @@ Function arguments are:
 - **`tlsVerify`** (defaults to `true`)
 
 
+### `nix2container.pullImageFromManifest`
+
+Pull a base image from a container registry using a supplied manifest file, and the
+hashes contained within it. The advantages of this over the basic `pullImage`:
+
+- Each layer archive is in its own store path, which means each will download just once
+  and naturally deduplicate for multiple base images that share layers.
+- There is no Nix-specific hash, so it's possible update the base image by simply
+  re-fetching the `manifest.json` from the registry; no need to actually pull the whole
+  image just to compute a new narhash for it.
+
+With this function the `manifest.json` functions as a lockfile meant to be stored in
+source control alongside the Nix container definitions.
+
```

Comment:
Sorry, one last thing: I think we should also explain to the user how to get the manifest, maybe with an example such as:
```nix
myImage = pullImageFromManifest { imageName = ...; imageManifest = ./manifest.json };
```

```bash
nix run .#myImage.getManifest > ./manifest.json
```

Reply:
Felt for now that it was easier to just leverage the one from the `examples` directory. so I went that way.